SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1647,3 +1647,21 @@ void USpatialNetDriver::DelayedSendDeleteEntityRequest(Worker_EntityId EntityId,
 		Sender->SendDeleteEntityRequest(EntityId);
 	}, Delay, false);
 }
+
+bool USpatialNetDriver::IsEntityListening(Worker_EntityId EntityId) const
+{
+	return ListeningEntities.Contains(EntityId);
+}
+
+void USpatialNetDriver::RegisterListeningEntity(Worker_EntityId EntityId)
+{
+	if (!ListeningEntities.Contains(EntityId))
```

Comment:
Does this need to be checked?

Reply:
Not really, it's safe to remove this check.

Reply:
Removed the check.